src/Organ.java

```diff
@@ -1,8 +1,12 @@
+import java.util.ArrayList;
+
 public abstract class Organ {
     private int currentCapacity, currentHealth, maxHealth, maxCapacity;
     private final int priority;
     private double resistance;
     private String name;
```

Comment:
What will the naming convention of organs be? Provide documentation on naming conventions.